app/models/manageiq/providers/kubernetes/container_manager/metrics_capture/hawkular_client_mixin.rb

```diff
@@ -1,6 +1,6 @@
 module ManageIQ::Providers::Kubernetes::ContainerManager::MetricsCapture::HawkularClientMixin
   def hawkular_client
-    require 'hawkular_all'
+    require 'hawkular/hawkular_client'
     @client ||= Hawkular::Metrics::Client.new(
       hawkular_entrypoint, hawkular_credentials, hawkular_options)
```

Comment:
No tenant option is passed when the client is created for the OpenShift context.